src/openrct2/ride/coaster/WoodenRollerCoaster.h

```diff
@@ -0,0 +1,79 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2022 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../../paint/Paint.h"
+#include "../../world/Location.hpp"
+#include "../TrackPaint.h"
+
+#include <cstdint>
+
+struct sprite_bb_2
+{
+    uint32_t sprite_id_a;
+    uint32_t sprite_id_b;
+    CoordsXYZ offset;
+    CoordsXYZ bb_offset;
+    CoordsXYZ bb_size;
+};
+
+template<bool isClassic> uint32_t wooden_rc_get_track_colour(const paint_session& session)
+{
+    if (isClassic)
+        return session.TrackColours[SCHEME_TRACK];
+    else
+        return (session.TrackColours[SCHEME_TRACK] & ~0xF80000) | session.TrackColours[SCHEME_SUPPORTS];
+}
+
+static uint32_t wooden_rc_get_rails_colour(paint_session& session)
```

Comment:
Debug seems to error here: `function 'wooden_rc_get_rails_colour' is not needed and will not be emitted [-Werror,-Wunneeded-internal-declaration]`

Reply:
Yeah, I read the error. What needs to be done to suppress it?